### PR TITLE
Implemented chat interface (UI only)

### PR DESCRIPTION
**Pull request by alanahodge**
Implemented UI for text-chat interface. Multiple hard-coded message examples are included for display/testing.

### CODE REVIEW

File: node_modules/react-chatbox-component/dist/style.css [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  margin: 25px auto 15px;
+}
+
+.empty-chat-holder img {
+  width: 100%;
+  height: 100%;
+}
+
+/* .avatar {
+  width: 50px;
```
Comment (NavyaGupta) [inline node_modules/react-chatbox-component/dist/style.css:35]: Remove?

File: node_modules/react-chatbox-component/dist/style.css [hunk 2]
```diff
@@ -0,0 +1,199 @@
+@import 'https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css';
+@import 'https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css';
+@import url('https://fonts.googleapis.com/css?family=Abril+Fatface&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```
Comment (NavyaGupta) [inline node_modules/react-chatbox-component/dist/style.css:4]: So for this we are using bootstrap lol? Is there a reason for this?

File: node_modules/react-chatbox-component/dist/style.css [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+.is-user {
+  background-color: #C8BFE7;
+}
+
+.is-other {
+  background-color: rgb(240, 235, 235);
+  -webkit-box-shadow: 0 2px 2px rgba(0, 0, 0, 0.1);
+  box-shadow: 0 2px 2px rgba(0, 0, 0, 0.1)
```
Comment (NavyaGupta) [inline node_modules/react-chatbox-component/dist/style.css:194]: What is this?

**Review by NavyaGupta — CHANGES_REQUESTED** [review]
Are we using bootstrap for this? If yes, I am not sure why we are using bootstrap for this when we have been using make styles. Is there a reason for this?

**Review by NerajM — CHANGES_REQUESTED** [review]
Your file structure is not consistent with the current structure of the codebase. You have to rebase and then commit again. You also have to solve the merge conflicts during the rebasing

File: node_modules/react-chatbox-component/dist/style.css (hunk 3 above)
Comment (bhawna-sharmaa) [reply node_modules/react-chatbox-component/dist/style.css:194]: It's what makes the users messages have a little shadow, just to give it a 3D look.

File: node_modules/react-chatbox-component/dist/style.css (hunk 2 above)
Comment (bhawna-sharmaa) [reply node_modules/react-chatbox-component/dist/style.css:4]: The bootstrap was leftover from a part we didn't need actually, but we just took that out.

File: node_modules/react-chatbox-component/dist/style.css (hunk 1 above)
Comment (bhawna-sharmaa) [reply node_modules/react-chatbox-component/dist/style.css:35]: Removed.

**Comment by alanahodge** [comment]
Removing PR, new ChatUI branch added.

**Pull request closed without merging**